includes/REST_API/Templates_Controller.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * Class Templates_Controller
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\REST_API;
+
+use stdClass;
+use WP_Error;
+use WP_Post;
+use WP_REST_Posts_Controller;
+use WP_REST_Request;
+use WP_REST_Response;
+
+/**
```

Comment:
Why is this controller different from `Stories_Controller`? `prepare_item_for_database` is a basic copy and pasted from that controller? Couldn't this controller just extend `Stories_Controller` class and remove part it doesn't need. It should keep the repeated logic down and the feature of the two controller similar but not the exactly the same. 

Either that, or create a new controller  called `Web_Stories_REST_Posts_Controller` and move repeated logic there.

Reply:
Yeah a new base controller makes sense.